src/grammar/query.rs

```diff
@@ -125,7 +398,11 @@ fn parse_column_expr(p: &mut Parser) {
     safe_loop!(p, {
         p.start(SyntaxKind::ColumnExpr);
 
-        parse_expr(p);
+        if p.nth(1) == Some(T!["("]) {
+            parse_function_invocation(p);
```

Comment:
A function may be schema qualified. I think we have a `opt_function_invocation` floating a round somewhere.

Either way, `parse_expr` also parses function invocations, so this here should not be necessary.

Reply:
Doesn't work with parse_expr only sadly. That leads to the same behaviour as before where the invocation is not detected

Reply:
Either parse_expr is bugged, or idk what's going on

Reply:
Was in fact because aggregate is a keyword and not an ident